test/test_parseconfig.py

```diff
@@ -46,12 +46,12 @@ def test_user_config(self):
 
     def test_project_config(self):
         project_config = self.config.read_project_config("./testfiles/zanata.xml")
-        self.assertEqual(project_config['project_url'], "http://localhost:8080/zanata/")
+        self.assertEqual(project_config['url'], "http://localhost:8080/zanata/")
         self.assertEqual(project_config['project_id'], "test-project")
         self.assertEqual(project_config['project_version'], "1.0")
-        self.assertEqual(project_config['locale_map'], {"zh-CN": "zh-Hans"})
-        self.assertEqual(project_config['src_dir'], "/home/user/project/source")
-        self.assertEqual(project_config['trans_dir'], "/home/user/project/target")
+        # self.assertEqual(project_config['locale_map'], {"zh-CN": "zh-Hans"})
```

Comment:
Actually, need to comment `locale_map` here https://github.com/sundeep-co-in/zanata-python-client/blob/STAGE/test/testfiles/zanata.xml#L7 so, that context can go for `remote locale_map` https://github.com/sundeep-co-in/zanata-python-client/blob/STAGE/zanataclient/context.py#L247

As, both `test_parseconfig` and `test_context` https://github.com/sundeep-co-in/zanata-python-client/blob/STAGE/test/test_context.py#L40 pointing to same test file `testfiles/zanata.xml`, need to disable that.